charts/datasaur/templates/config/grafana-alert-configMap.yaml

```diff
@@ -3,4 +3,9 @@ kind: ConfigMap
 metadata:
   name: grafana-alert-provisioning
 data:
-{{ .Values.loki.alerting | toYaml | indent 1 }}
\ No newline at end of file
+{{- if .Values.loki.alerting }}
```

Comment:
kenapa nama fieldnya `alerting`?

Reply:
mau di ganti apa? xD @tonoman3g

Reply:
oh, gw baru ngeh, gpp **alerting** aja

di grafana semuanya refer ke term alerting

https://grafana.com/blog/2021/06/14/the-new-unified-alerting-system-for-grafana-everything-you-need-to-know/